pkg/bff/config/config.go

```diff
@@ -43,6 +43,7 @@ type Config struct {
 	Database     config.StoreConfig
 	Email        EmailConfig
 	Sentry       sentry.Config
+	UserCache    CacheConfig `split_words:"true"`
```

Comment:
For some reason the variables were not being picked up from the environment without this tag.

Reply:
Aha, thanks for clearing that up.